src/Model/ExtDate.php

```diff
@@ -175,6 +175,27 @@ private function resolveMaxDay( $year, $month ): int {
 		return null === $this->day ? $lastDayOfMonth : $this->day;
 	}
 
+	public function getUnspecifiedYearScale() : int {
+		if ( $this->unspecifiedDigit->unspecified( 'year' ) ) {
+			$ret = $this->unspecifiedDigit->getYear();
+
+			if ( $this->getYear() === 0 ) {
+				return $ret;
+			}
+
+			return $ret + 1;
+		}
+		return 0;
```

Comment:
Can apply https://www.entropywins.wtf/blog/2019/01/14/readable-functions-guard-clause/

`$ret` is not a good var name